fjord/feedback/templates/feedback/generic_feedback.html

```diff
@@ -149,27 +149,14 @@
               {# NOTE: "Firefox" only until we figure out how to deal with browser/product mismatches. #}
               <div id="browser-ask" class="private">
                 <div class="ask">
-                  <input id="browser-ok" name="browser_ok" type="checkbox">
+                  <input id="browser-ok" name="browser_ok" type="checkbox" checked>
                   <label class="browser-ok" for="browser-ok">
-                    <span class="happy">
-                      {% trans %}
-                        (Optional)
-                        Check here to provide technical information
-                        about your browser. This information helps us
-                        better understand our users and create better
-                        products.
-                        This data will be kept private.
-                      {% endtrans %}
-                    </span>
-                    <span class="sad">
-                      {% trans %}
-                        (Optional)
-                        Check here to provide technical information
-                        about your browser. This will help us diagnose
-                        problems more quickly.
-                        This data will be kept private.
-                      {% endtrans %}
-                    </span>
+                    {% trans %}
+                      Check here to provide technical information about your browser.
+                      This information helps us better understand our users and create better
+                      products. This information will be kept in accordance with our Privacy
+                      policy.
+                    {% endtrans %}
```

Comment:
We no longer have happy/sad text--it's just one text for all situations. This was the text agreed upon by privacy, legal, my mom, a fish sandwich I thought I finished the other day and probably some other people.